source/extensions/extensions_metadata.yaml

```diff
@@ -750,3 +750,73 @@ envoy.http.stateful_session.cookie:
   - envoy.http.stateful_session
   security_posture: unknown
   status: alpha
+envoy.matching.inputs.request-headers:
```

Comment:
Thanks, since we're going to do rename anyway, lets use `snake_case` all the case here?

Reply:
Done.

Reply:
GH is down, can't push the commit.